packages/install-expo-modules/src/plugins/ios/withIosModulesPodfile.ts

```diff
@@ -36,8 +36,27 @@ export function updatePodfile(
     contents = contents.replace(targetRegExp, '$1\n  use_expo_modules!');
   }
 
+  // use_native_modules! from expo-modules-autolinking
+  if (sdkVersion && semver.gte(sdkVersion, '52.0.0')) {
+    const autolinkinRegExp = /^(\s+config\s+=\s+use_native_modules!\s*)$/m;
+    const newBlock = `\
+  config_command = [
+    'node',
+    '--no-warnings',
+    '--eval',
+    'require(require.resolve(\\'expo-modules-autolinking\\', { paths: [require.resolve(\\'expo/package.json\\')] }))(process.argv.slice(1))',
+    'react-native-config',
+    '--json',
+    '--platform',
+    'ios'
+  ]
+  config = use_native_modules!(config_command)
```

Comment:
might need to be updated, depending on https://github.com/expo/expo/pull/32161#discussion_r1809254536